docker/local/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Meant to be run from the root of the repo
+
+docker build -f ./docker/local/Dockerfile \
+  --build-context clickhouse=./docker/clickhouse \
+  --build-context otel-collector=./docker/otel-collector \
+  --build-context ingestor=./docker/ingestor \
+  --build-context local=./docker/local \
+  --build-context api=./packages/api \
+  --build-context app=docker-image://ghcr.io/hyperdxio/hyperdx:1.3.0-app \
```

Comment:
Is this intentional ? (app context)

Reply:
Yes so we can avoid adding a builder step in the already tiring docker file 😅

Reply:
that's true - now that i typed it out, i'll probably need to add the builder step here anyways in this dockerfile since we'll need to have a specific bundle for local mode for a few small changes. so for now this is really just a placeholder.